### PR TITLE
collapsible schedule blocks

### PR DESCRIPTION
**Pull request by ryanpitts**
This should be tested pleeeeeease :)

Here's a possible implementation for @cassiemc's open ticket https://github.com/mozilla/mozfest-schedule-app/issues/52. Adds collapsible schedule blocks on the daily schedule tabs using CSS transitions, and a "collapse/expand" link at top that triggers all blocks on the page. Needs attention to icons, link colors, etc.

It is working for me, but I'm on my laptop in the airport waiting to get on a flight and can't really test beyond what I'm seeing right now.

If this implementation seems like a bad idea, please kill this PR!

### CODE REVIEW

File: js/schedule.js [hunk 1]
```diff
@@ -670,6 +688,44 @@ function Schedule(options) {
             }
         });
 
+        // toggle individual schedule blocks on header tap
+        schedule.$container.on('click', '.slider-control', function(e) {
+            var clicked = $(this);
+            var targetBlock = clicked.next('.page-block');
+            targetBlock.toggleClass('closed');
+            if (targetBlock.hasClass('closed')) {
```
Comment (mmmavis) [inline js/schedule.js]: wondering if we should extract the following if-else into an animation helper function so we can reuse it for "toggle all?

**Comment by mmmavis** [comment]
looking great! just one comment: https://github.com/mozilla/mozfest-schedule-app/pull/65/files#r44003458

We might want to tweak the wording for "Collapse all" and "Expand all" but we can tackle that in another PR. :+1: :smile:

File: js/schedule.js (hunk 1 above)
Comment (ryanpitts) [inline js/schedule.js]: yeah, I thought about that after I wrote the same thing twice, but then I was super tired and I went to bed :)

will extract and push an update

**Comment by mmmavis** [comment]
awesome! let' hit the green button now! :tada:

**Pull request merged**